inc/commonitilobject.class.php

```diff
@@ -5016,7 +5017,8 @@ static function getAllTypesForHelpdesk() {
 
       // Drop not available plugins
       foreach (array_keys($ptypes) as $itemtype) {
-         if (!in_array($itemtype, $_SESSION["glpiactiveprofile"]["helpdesk_item_type"])) {
+         if (isset($_SESSION["glpiactiveprofile"]["helpdesk_item_type"])
+             && !in_array($itemtype, $_SESSION["glpiactiveprofile"]["helpdesk_item_type"])) {
```

Comment:
It is an exclude list, so maybe the check should consider that no types are available if list is not set.
```suggestion
         if (!isset($_SESSION["glpiactiveprofile"]["helpdesk_item_type"])
             || !in_array($itemtype, $_SESSION["glpiactiveprofile"]["helpdesk_item_type"])) {
```